timemachine/fe/model_rabfe.py

```diff
@@ -97,11 +97,12 @@ def simulate_futures(
             bp = unbound_pot.bind(np.asarray(params))
             bound_potentials.append(bp)
 
+        # Always get at least one sample
+        subsample_interval = min(self.prod_steps, 1000)
```

Comment:
These changes are gross, but seemed like the simplest way to ensure that you get at least one frame back

Reply:
Another alternative is that if the interval is greater than the number of steps, reduce the interval to the number of steps